ersilia/cli/commands/catalog.py

```diff
@@ -80,6 +86,21 @@ def catalog(
             mc.airtable()
             return
         catalog_table = mc.local() if local else mc.hub()
+        if local is True and browser is True:
```

Comment:
This code from L89 to L102 is repeating, ie it's the same as code from L75 till L88

Reply:
I must have missed that while resolving the merge conflict, I'll fix this